po/fa.po

```diff
@@ -3747,21 +3879,34 @@ msgid ""
 "i16` implementation automatically create an implementation of `Into<i16> for "
 "i8`."
 msgstr ""
+"کتابخانه استاندارد پیاده‌سازی <span dir=ltr>`From<i8> i16`</span> به نوع "
```

Comment:
Please update the rest of paragraph yourself, it's a little hard to review this bidi lines in github.
```suggestion
"کتابخانه استاندارد یک پیاده‌سازی <span dir=ltr>`From<i8> for i16`</span> دارد "
```